README.md

```diff
@@ -1,45 +1,146 @@
+
+#### Dev Dependencies
+
+- [nodemon](https://ghub.io/nodemon): Simple monitor script for use during development of a node.js app.
+
+## Requirements
+
+The catapult-explorer works on NodeJS versions:
+
+- 8.9.X
+- 9.X.X
+- 10.16.X
+
+## Installation
+
+1. Install the client required dependencies.
+
 ```
 cd client
+
 npm install
 ```
 
-#### Compiles and hot-reloads for development
+2. Install the server required dependencies.
+
 ```
-cd client
-npm run serve
+cd server
+
+npm install
+
+cp .env.example .env
+
 ```
 
-#### Compiles and minifies for production
+## Usage
+
+1. Run the client application.
+
 ```
 cd client
-npm run build
+npm run serve
 ```
 
-#### Lints and fixes files
+2. Open a new terminal and run the server.
+
 ```
 cd client
-npm run lint
+npm start
 ```
 
-## Setup API Server
-```
-cd server
-npm install
-```
+3. Visit http://localhost:8080 in your browser.
 
-#### Setup environment
-```
-cp .env.example .env
+## Contributing
 
-** Configure all required variables in .env
-```
+This project is developed and maintained by NEM Foundation. Contributions are welcome and appreciated. You can find [catapult-explorer on GitHub][self];
+Feel free to start an issue or create a pull request. Check [CONTRIBUTING](CONTRIBUTING.md) before start.
 
-#### Compiles and hot-reloads for development
-```
-cd server
-npm start
-```
+### Submitting your changes
+
+Once your changes and tests are ready to submit for review:
+
+1.  Test your changes.
+
+    If you added a new component in `Client` or new features in `Server`:
+
+    please follow the step [here](https://github.com/nemfoundation/catapult-explorer#usage), ensure that is no error message, and your change or feature is working.
+
+2.  Submit a pull request.
+
+    Push your local changes to your forked copy of the repository and [submit a pull request](https://help.github.com/articles/about-pull-requests/). In the pull request, choose a title which sums up the changes that you have made, and in the body provide more details about what your changes do. Also mention the number of the issue where discussion has taken place, eg "Closes #123".
+
+Then sit back and wait. There will probably be a discussion about the pull request and, if any changes are needed, we would love to work with you to get your pull request merged into catapult-explorer.
+
+_CONTRIBUTING.md is based on [CONTRIBUTING-template.md](https://github.com/nayafia/contributing-template/blob/master/CONTRIBUTING-template.md)_ , [elasticsearch/CONTRIBUTING](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md) and [spark/CONTRIBUTING](https://github.com/apache/spark/blob/master/CONTRIBUTING.md)
+
+[pull-request]: https://help.github.com/articles/about-pull-requests/
+[github-issues]: https://github.com/nemfoundation/catapult-explorer/issues
+
+## Getting help
+
+We use GitHub issues for tracking bugs and have limited bandwidth to address them.
+Please, use the following available resources to get help:
+
```

Comment:
You are not listing any resource, probably link to the docs (NEM Developer Center) or redirect to Slack.